docs/modules/ROOT/pages/shipping-logs-to-amazon-elasticsearch-service.adoc

```diff
@@ -1,71 +1,102 @@
+
 :url-github-elasticsearch: https://github.com/elastic/elasticsearch
-:url-amazon-elasticsearch: https://aws.amazon.com/elasticsearch-service
+:url-amazon-opensearch: https://aws.amazon.com/opensearch-service
+:url-aws-signup: https://portal.aws.amazon.com/billing/signup 
 :url-aws-console: https://us-west-2.console.aws.amazon.com/es/home
 :page-dedicated: true
 
-Amazon provides link:{url-github-elasticsearch}[ElasticSearch] as a service which is called link:{url-amazon-elasticsearch}[ElasticSearch Service]. You can create ElasticSearch services with some built-in plugins like Kibana. In this tutorial, you will see how easy it is to set up an ElasticSearch service on AWS then ship Hazelcast Cloud cluster logs to ElasticSearch. 
-
-== Set Up an ElasticSearch Service Instance
-
-Go to link:{url-aws-console}[] and click *Create a new domain*
-
-image:elasticsearch.png[Homepage for Amazon ElasticSearch Service]
-
-On the initial page of domain creation, you can select your deployment type. For testing purpose, you can select *Development and Testing*, or if it is a live environment you can select *Production* option, *Custom* otherwise. And *7.7* is recommended for Elasticsearch version.
-
-image:elasticsearch-deployment-type.png[A list of deployment types to choose from while creating a domain]
-
-On the next screen, you can configure your Elasticsearch instance's specs; domain name, machine type, and EBS volume size. 
-
-image:elasticsearch-configure-domain.png[Configuration options for a new domain, including name, instance type and number of nodes]
+{description}
 
-Once you configured specs, you can continue with the security part which is the most critical one. It is not possible to use *VPC Access* if you will ship logs from Hazelcast Cloud since there should be a VPC Peering between ES instance and Hazelcast Cloud Cluster VPC, at least there is no support for now. Select *Public Access* and scroll down below configurations.
+link:{url-amazon-elasticsearch}[Amazon OpenSearch Service] is based on link:{url-github-elasticsearch}[Elasticsearch], and allows you to search, analyze and visualize your cluster logs using Kibana.
 
-image:elasticsearch-configure-security.png[Security configuration options, including public internet access or VPC access]
+[[set-up-opensearch-domain]]
+== Set Up an Amazon OpenSearch Domain
 
-Since your ES instance has public access, you need to apply Fine-Grained access control to allow certain users to access. Select *Create master user* and provide username/password pair. 
+Create an Amazon OpenSearch domain to receive log files from your Hazelcast Cloud cluster.
 
-NOTE: Save Username / Password pair to a safe place to use it Hazelcast Cloud logging configuration in next steps.
+. link:{url-aws-signup}[Create an AWS account] if you don't have one already.
+. Sign in as root user.
+. From the list of regions on the top menu, select *US West (N California) us-west-1*.
```

Comment:
The advice is to choose a region that's closest to your Hazelcast cluster to reduce latency.

Reply:
Updated as requested.